applications/luci-app-acme/Makefile

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2017 Dan Luedtke <mail@danrl.com>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=ACME package - LuCI interface
+LUCI_DEPENDS:=luci-base luci-app-uhttpd +acme
```

Comment:
I think that the dependencies should be selective. Currently the app is well hidden, as it is only visible if you have installed luci-app-uhttpd

`LUCI_DEPENDS:=+luci-base +luci-app-uhttpd +acme`